pods/main-chart-list/main-chart-list.api.ts

```diff
@@ -0,0 +1,9 @@
+import { ChartInfoVm } from "./main-chart.vm";
+import { mainChartList } from "./main-chart-list.data";
+import { of } from "rxjs";
+import { delay } from "rxjs/operators";
+
+const ONE_SECOND: number = 1000;
+
+export const getAllCharts = (): Promise<ChartInfoVm[]> =>
+  of(mainChartList).pipe(delay(ONE_SECOND)).toPromise();
```

Comment:
In this case we don't need to simulate a promise, our first version will go live with a list obtained directly from the client side, once we reach something like 50 chart info we will move it to server and start thinking about a DB